src/material/schematics/ng-update/test-cases/v9/hammer-migration-v9.spec.ts

```diff
@@ -310,9 +313,13 @@ describe('v9 HammerJS removal', () => {
         }
       `);
 
-      await runMigration();
+      const {logOutput} = await runMigration();
 
       expect(tree.readContent('/projects/cdk-testing/src/main.ts')).toContain(`import 'hammerjs';`);
+      expect(logOutput).toContain(
+        'General notice: The HammerJS v9 migration for Angular components is not able to ' +
+        'migrate tests. Please manually clean up tests in your project if they rely on the ' +
+        'deprecated Angular Material gesture config.');
```

Comment:
Can we link them to the gesture config API or docs in case they haven't touched that part of their app in years or no one who set that up is still on the team?

If you Google "Angular Material gesture config", you mostly just see docs for Angular's `HammerGestureConfig` and the Angular Material Getting Started Guide. So that won't be too helpful to them if we can't provide them some additional context/reference here.

Reply:
This message will be only printed if they explicitly reference the `GestureConfig` from Angular Material. I think we can make the assumption that it is clear enough in that case.

I'm hesitant to adding any link since our docs page is likely to change in the future and we don't even have docs for `material/core` IIRC.

Reply:
It would be nice to at least mention the `GestureConfig` class explicitly in that case as some people may not immediately know that "gesture config" maps directly to that class.